### ISSUE TITLE
[UWP][Accessibility] Name property of a focusable element is null.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3850]
# Platform

* UWP

# Version of SDK

master

# Details

​## Repro Steps:​
1. Launch the application.
2. New screen starts appearing. Navigate to Left pane JSON files.​
3. Now select "FlightDetails.Json" button. It will update the Middle and Right Pane.
4. Open Accessibility Insights For Windows.
5. Verify Name property of Coding section element.

## Actual Result:
Name property of a focusable element is null and due to this, screen reader only announces it as 'edit'. Hence, user won't get the purpose of such element or control.

## Expected Result:
Name property should be defined for Coding section element under  FlightDetails.Json file.
